makeflow/src/dag_node.h

```diff
@@ -72,9 +83,19 @@ struct dag_node {
 	time_t previous_completion;
 
 	struct dag_node *next;              /* The next node in the list of nodes */
+
+	int updated;						/* Int indicating this node has updated its size */
+};
+
+struct dag_node_size {
+	struct dag_node *n;
+	uint64_t size;
```

Comment:
Why do we have another structure for this? Isn't it easier to add size as a field of node?